libs/clients/air-discount-scheme/src/lib/air-discount-scheme.config.ts

```diff
@@ -0,0 +1,27 @@
+import { defineConfig } from '@island.is/nest/config'
+import * as z from 'zod'
+
+const schema = z.object({
+  tokenExchangeScope: z.array(z.string()),
+  timeout: z.number(),
+  basePath: z.string(),
+})
+
+export const AirDiscountSchemeClientConfig = defineConfig<
+  z.infer<typeof schema>
+>({
+  name: 'AirDiscountSchemeClient',
+  schema,
+  load(env) {
+    return {
+      tokenExchangeScope: env.optionalJSON(
+        'AIR_DISCOUNT_SCHEME_CLIENT_SCOPE',
+      ) ?? ['@vegagerdin.is/air-discount-scheme-scope'],
+      timeout: env.optionalJSON('AIR_DISCOUNT_SCHEME_CLIENT_TIMEOUT') ?? 20000,
+      basePath: env.required(
+        'AIR_DISCOUNT_SCHEME_BACKEND_URL',
+        'http://localhost:4248',
```

Comment:
Is this correct?

Reply:
Nice! 
I forgot to add `AIR_DISCOUNT_SCHEME_CLIENT_TIMEOUT` to the environment but I generated `AIR_DISCOUNT_SCHEME_BACKEND_URL` for the charts accordingly, to the best of my knowledge.